lr

```diff
@@ -804,7 +804,11 @@ print "\nEXECUTING THE CHECK POWER COMMAND PROGRAM\n";
 # Now execute Check_Power_Cmds.py
 system("/proj/sot/ska3/flight/bin/python /data/acis/LoadReviews/script/CHECK_POWER_COMMANDS/Check_Power_Cmds.py");
 
-# Done with Check_Power_Cmds.py so go back to $lr_dir
+print "\nEXECUTING THE WINDOW CHECK PROGRAM.\n";
+# Now execute the Window Check program
+system('python3 /data/acis/LoadReviews/script/WINDOW_CHECK/Window_Check.py');
```

Comment:
I'm curious of the reason you are using whatever `python3` is in the current path instead of the configured Ska3 python?

Reply:
Because we want this tool to be able to run anywhere on any machine - even those who do not have access to Ska3.  Especially important these days when we might be doing load reviews from home.  There is no use of any tool in  Ska3 so there is no need for limiting execution to only machines that have access to Ska3.

Reply:
Given that `Window_Check.py` includes hardwired references to `/home/gregg/SI_MODES` and to `/data/acis/LoadReviews/`, at first glance it does not seem this script is going to be portable.   And of course `lr` itself does require ska3 based on the `system` call two lines before.  

Anyway, the main concern is that the configuration used for testing is the same as that used for flight review.  Using `python3` leaves open the possibility that a different version of Python is in the path and the tool is run in an untested configuration.  In practice that probably does not happen, but the code would appear to allow that.

Reply:
The reference to /home/gregg/SI_MODES is a good catch - thanks. 

The reference to /home/gregg/SI_MODE in the si_mode_dat_dir assignment in Window_Check.py  is superfluous - it was used for testing and development  and isn't actually used. So that has been removed.

While the default for the call to RunRat is /home/gregg/...it is not used that way in the code. It is called with  hard coded /data/acis/LoadReviews......   However the default has been set to:
   
/data/acis/LoadReviews/script/WINDOW_CHECK/LCMDs

same as the call.

These changes are DONE and pushed up into PR #25 


ACIS Ops LR code must be run with a /data/acis/LoadReviews....... and /data/acis/sacgs... directory structures and contents.  But this structure can be simply copied on any machine. No special or complicated installations are required.  And in fact we've done so already in order to investigate what it would take to run LR out of Wayside (or anywhere else)  with no network connections.  I've also done this on my home linux box.

Being self-contained is a goal-in-work for LR software  - and has become more urgent given current events - so that it can be executed anywhere, including home machines of both SAO and MIT ACIS Ops users with  minimum of outside connection/dependencies.  I am reviewing all LR tools with this goal in mind. 

Standard operation of LR uses HEAD installations for things like Python. When Systems changes it we need to run regression tests. Ska3 Python is no less susceptible to changes than HEAD and that, too, requires that re-test.